recipe/meta.yaml

```diff
@@ -1,71 +1,129 @@
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
     - m2-patch  # [win]
     - patch  # [not win]
-  host:
-    - python
-    - cython
-    - geos
-    - numpy
-    - pip
-    - proj
-    - pyproj
-    - setuptools
-    - wheel
-  run:
-    - python
-    - {{ pin_compatible('numpy') }}
-    - geos
-    - matplotlib-base >=1.0.0,!=3.0.1
-    - proj
-    - pyproj >=1.9.3
-    - pyshp >=1.2.0
-    - six
 
-test:
-  imports:
-    - mpl_toolkits.basemap
-  requires:
-    - pip
-  commands:
-    - pip check
-    - python -c "from mpl_toolkits.basemap import Basemap"
 
-about:
-  home: http://matplotlib.org/basemap
-  license: MIT
-  license_family: MIT
-  license_file: README.md
-  summary: 'Plot on map projections using matplotlib'
-  description: |
-    Basemap toolkit is a library for plotting 2D data on maps in Python.
-    It is similar in functionality to the matlab mapping toolbox, the IDL mapping facilities, GrADS, or the Generic Mapping Tools.
-  dev_url: https://github.com/matplotlib/basemap
-  doc_url: http://matplotlib.org/basemap/
-  doc_source_url: https://github.com/matplotlib/basemap/blob/master/doc/index.rst
+outputs:
+
+  - name: basemap
+    script: build-basemap.sh  # [not win]
+    script: bld-basemap.bat   # [win]
+    build:
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - python
+        - pip
+        - cython >=0.29,<3.1
+        - numpy >=1.21  # [not ppc64le]
+        - numpy >=1.19  # [ppc64le]
+        - geos
+        - pyproj
+        - setuptools
+        - wheel
+      run:
+        - {{ pin_subpackage('basemap-data') }}
+        - python
+        - {{ pin_compatible('numpy') }}
+        - {{ pin_compatible('geos') }}
+        - matplotlib-base >=1.5,!=3.0.1
+        - pyproj >=1.9.3
+        - pyshp >=1.2.0
+        - six >=1.10,<1.16
+    test:
+      imports:
+        - mpl_toolkits.basemap
+      requires:
+        - pip
+        - {{ pin_subpackage('basemap-data') }}
+      commands:
+        - pip check
+        - python -c "from mpl_toolkits.basemap import Basemap"
+    about:
+      home: http://matplotlib.org/basemap
+      license: MIT
+      license_family: MIT
+      license_file:
+        - packages/basemap/LICENSE
+      summary: 'Plot on map projections using matplotlib'
+      description: |
+        Basemap toolkit is a library for plotting 2D data on maps in Python.
+        It is similar in functionality to the matlab mapping toolbox, the IDL mapping facilities, GrADS, or the Generic Mapping Tools.
+      dev_url: https://github.com/matplotlib/basemap
+      doc_url: http://matplotlib.org/basemap/
+      doc_source_url: https://github.com/matplotlib/basemap/blob/v{{ version }}/packages/basemap/doc/index.rst
+
+  - name: basemap-data
+    script: build-basemap.sh  # [not win]
+    script: bld-basemap.bat   # [win]
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.6
```

Comment:
A plain `python` should be  in `host` without pinning

Reply:
Fixed: https://github.com/AnacondaRecipes/basemap-feedstock/blob/sb_update_v1.3.1/recipe/meta.yaml#L86